po/es.po

```diff
@@ -6755,48 +6826,48 @@ msgid ""
 "With the first call to `say_hello`, `main` gives up ownership of `name`. "
 "Afterwards, `name` cannot be used anymore within `main`."
 msgstr ""
-"Con la primer llamada a `say_hello`, `main` da propiedad de `name`. Después "
-"de todo, `name` no puede usarse meas dentro de `main`."
+"Con la primera llamada a `say_hello`, `main` deja de tener la _ownership_ de "
+"`name`. Después, ya no se podrá usar `name` dentro de `main`."
 
 #: src/ownership/moves-function-calls.md:21
 msgid ""
 "The heap memory allocated for `name` will be freed at the end of the "
 "`say_hello` function."
 msgstr ""
-"La pila de memoria se asigna para `name` y será liberada al final de la "
-"función `say_hello`."
+"La memoria de _heap_ asignada a `name` se liberará al final de la función "
+"`say_hello`."
 
 #: src/ownership/moves-function-calls.md:22
 msgid ""
 "`main` can retain ownership if it passes `name` as a reference (`&name`) and "
 "if `say_hello` accepts a reference as a parameter."
 msgstr ""
-"`main` puede mantener la propiedad si pasa `name` como referencia (`&name`) "
-"y si `say_hello` acepta una referencia como un parámetro."
+"main` podrá conservar la _ownership_ si pasa`name` como referencia (`&name`) "
+"y si `say_hello` acepta una referencia como parámetro."
 
 #: src/ownership/moves-function-calls.md:23
 msgid ""
 "Alternatively, `main` can pass a clone of `name` in the first call (`name."
 "clone()`)."
 msgstr ""
-"Alternativamente, `main` puede pasar un clon de `name` in la primer llamada "
+"Por otro lado, `main` puede pasar un clon de `name` en la primera llamada "
 "(`name.clone()`)."
 
 #: src/ownership/moves-function-calls.md:24
 msgid ""
 "Rust makes it harder than C++ to inadvertently create copies by making move "
 "semantics the default, and by forcing programmers to make clones explicit."
 msgstr ""
-"Rust hace esto más duro que C++ para crear copias automáticas mediante mover "
-"semánticas por defecto, y forzando a los programadores a hacer clones "
-"explícitos."
+"Rust hace que resulte más difícil que en C++ crear copias por error al "
+"definir la semántica de movimiento como predeterminada y al obligar a los "
+"programadores a hacer clones explícitos."
```

Comment:
"hacer clones explícitos" -> seems to suggest that whatever you cloned it is explicit somehow, like, not the action of cloning is explicit, but the result is explicit. This needs rewording.

"a hacer clones de forma explícita"
"a explícitamente hacer clones"
"a hacer clones explícitamente"

I'm unsure. The other problem here  is the verb "hacer" which seems to get in way of conveying the actual meaning.

What about:

"obligar a los programadores a clonar sólo de forma explícita."